staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -148,11 +149,13 @@ func (pw *prefixWriter) Write(level int, format string, a ...interface{}) {
 	for i := 0; i < level; i++ {
 		prefix += levelSpace
 	}
-	fmt.Fprintf(pw.out, prefix+format, a...)
+	output := fmt.Sprintf(prefix+format, a...)
```

Comment:
`fmt.Sprintf` automatically escapes special characters?

Reply:
This formats it into a string, where WriteEscaped on the next line does that.